sdk/translation/Azure.AI.Translation.Document/README.md

```diff
@@ -27,11 +27,11 @@ dotnet add package Azure.AI.Translation.Document
 Document Translation supports [single-service access][single_service] only.
 To access the service, create a Translator resource.
 
-You can create either resource using:
+You can create either resource using: 
```

Comment:
I see a few changes like this one where the only difference is the extra whitespace. Consider reverting those changes.

Reply:
Fixed & pushed.